src/Libuv/project.json

```diff
@@ -15,7 +15,7 @@
     ],
     "files": {
       "mappings": {
-        "/": "../../content/License.txt",
+        "/": "License.txt",
```

Comment:
I assume you are referring to Microsoft.NETCore.Platforms": "1.1.0",. It does not have to change - should it stay at 1.0.1?